drivers/at/at.c

```diff
@@ -22,11 +22,16 @@
 #define AT_PRINT_INCOMING (0)
 #endif
 
+static void isrpipe_write_one_wrapper(void *_isrpipe, uint8_t data)
```

Comment:
Maybe inline this?

Reply:
Even if I mark it as inline, the compiler won't be able to inline it because line 34 below needs a pointer to this function. That is why @A-Paul 's experiment failed:

> Code size is increasing by 8 byte. I tried inline but gained 8 bytes too.

Reply:
Doh, you're right :coffee:

Reply:
As it is a static function, is not usual to write it starting with an `_` ?

Reply:
yes, auto-suggesting  change:
```suggestion
static void _isrpipe_write_one_wrapper(void *_isrpipe, uint8_t data)
```